packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -498,8 +498,13 @@ const ssr = (() => {
     if (mode === production) {
         return baseConfig('node')
             .extend((config) => {
+                let additionalConfig = {}
+                if (process.env.SSR_SOURCE_MAP === 'true') {
+                    additionalConfig = {devtool: 'source-map'}
+                }
                 return {
                     ...config,
+                    ...additionalConfig,
```

Comment:
nit: you don't really need to build up and spread a whole new object for just one property...
```suggestion
                    devtool: process.env.SSR_SOURCE_MAP === 'true' ? 'source-map' : undefined,
```
Depending on how webpack handles `devtool: undefined`, you might replace that with some other value. Or, if it must be omitted, just inline the spread:
```suggestion
                    ...(process.env.SSR_SOURCE_MAP === 'true' ? {devtool: 'source-map'} : {}),
```

Reply:
Gotcha, thank you!